hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/events/SCMEvents.java

```diff
@@ -157,6 +157,13 @@ public final class SCMEvents {
   public static final TypedEvent<DatanodeDetails> NEW_NODE =
       new TypedEvent<>(DatanodeDetails.class, "New_Node");
 
+  /**
+   * This event will be triggered whenever a datanode is registered with
+   * SCM with a different Ip or host name.
+   */
+  public static final TypedEvent<DatanodeDetails> NODE_IP_OR_HOSTNAME_UPDATE =
+          new TypedEvent<>(DatanodeDetails.class, "Node_Ip_Or_Hostname_Update");
```

Comment:
Instead of "IP or hostname" I think we can simply say "address", here and elsewhere, too (e.g. in `NodeIpOrHostnameUpdateHandler`).

Reply:
sure.